src/styles/index.scss

```
@@ -1,3 +1,130 @@
+@import './mixins';
+
+$content-width: 166px;
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.cards {
+  border: 1px solid #f3f3f3;
```

add padding to a card
```suggestion
  padding: 32px 16px 16px;
  border: 1px solid #f3f3f3;
```

---

fixed